Couldn't add a player to the whitelist

[17:24:35 WARN]: [VoidWhitelist] Couldn't add a player to the whitelist
java.util.concurrent.CompletionException: java.lang.NullPointerException: expiresAt is marked non-null but is null
        at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
        at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
Caused by: java.lang.NullPointerException: expiresAt is marked non-null but is null
        at voidpointer.spigot.voidwhitelist.storage.SimpleWhitelistable.<init>(SimpleWhitelistable.java:29) ~[VoidWhitelist-1.0.0b480-stable.jar:?]
        at voidpointer.spigot.voidwhitelist.storage.MemoryWhitelistService.lambda$add$4(MemoryWhitelistService.java:109) ~[VoidWhitelist-1.0.0b480-stable.jar:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        ... 6 more

--- a/pom.xml
+++ b/pom.xml
@@ -21,7 +21,7 @@
 
     <groupId>voidpointer.spigot</groupId>
     <artifactId>voidwhitelist</artifactId>
-    <version>1.0.0</version>
+    <version>1.0.1</version>
     <packaging>jar</packaging>
 
     <name>VoidWhitelist</name>

--- a/src/main/java/voidpointer/spigot/voidwhitelist/storage/SimpleWhitelistable.java
+++ b/src/main/java/voidpointer/spigot/voidwhitelist/storage/SimpleWhitelistable.java
@@ -33,7 +33,6 @@ public final class SimpleWhitelistable extends AbstractWhitelistable {
     private UUID uniqueId;
     @NonNull
     private String name;
-    @NonNull
     private Date expiresAt;
 
     @Override public boolean isAssociatedWith(final Player player) {

